fixtures/test.ini

```diff
@@ -10,7 +10,7 @@ time_start_regex = IRTIT3_(?P<time_coverage_start>\d{8})_
 pixel_size = 25
 time_coverage_duration = P0DT23H59M59S
 browse_regex = _brws
-granule_regex = _(?P<granuleid>[NS]{1}\d{2}km_\d{8})_
+granule_regex = (NSIDC0081_SEAICE_PS_)(?P<granuleid>[NS]{1}\d{2}km_\d{8})(_v2.0_)(?:F\d{2}_)?(DUCk)
```

Comment:
This is only meant to be an example regex. The `test.ini` is now a mishmash of values for different collections.